pkg/tagger/collectors/kubelet_extract.go

```diff
@@ -90,7 +90,13 @@ func (c *KubeletCollector) parsePods(pods []*kubelet.Pod) ([]*TagInfo, error) {
 			case "StatefulSet":
 				tags.AddLow("kube_stateful_set", owner.Name)
 			case "Job":
-				tags.AddOrchestrator("kube_job", owner.Name) // TODO detect if no from cronjob, then low card
+				cronjob := c.parseCronJobForJob(owner.Name)
+				if cronjob != "" {
+					tags.AddOrchestrator("kube_job", owner.Name)
+					tags.AddLow("kube_cronjob", cronjob)
+				} else {
+					tags.AddLow("kube_job", owner.Name)
```

Comment:
Is it supposed to be `AddOrchestrator` or `AddLow` on `kube_job`?

Reply:
AddLow if the job is not created by a cronjob, AddOrchestrator otherwise. My reasoning was that the job name changes every time if it's created by a cronjob, so that could be high-ish cardinality. And since all these jobs are the same (they come out from the same cronjob) we can omit their name and aggregate up. No such guarantee with "free" job, so we have to attach the job name every time.

Does that sound reasonable?